src/context/mod.rs

```diff
@@ -829,6 +830,36 @@ impl Context {
             .map_err(|_e| ValkeyError::Str("User does not have permissions on key"))
     }
 
+    pub fn get_client_id(&self) -> u64 {
```

Comment:
@KarthikSubbarao - also since mod.rs is a very large file I am going to move these to src/context/client.rs and do `impl Context` for these 5 functions.